dftimewolf/lib/containers/containers.py

```diff
@@ -669,6 +669,30 @@ def __str__(self) -> str:
     return self.description
 
 
+class SQLQuery(interface.AttributeContainer):
+  """SQL Query container.
+
+  Attributes:
+    query: The SQL query string.
+    description: A description of the query.
+  """
+
+  CONTAINER_TYPE = "sql_query"
+
+  def __init__(self, query: str, description: str) -> None:
+    super(SQLQuery, self).__init__()
```

Comment:
```suggestion
    super().__init__()
```
Python 3 doesn't require explicit arguments.